components/js/override.js

```diff
@@ -22,13 +22,10 @@ define(['angular'], function(angular) {
     .constant('OVERRIDE', {
       'APP_OPTIONS': {
         'appMenuTemplateURL': 'portal/misc/partials/example-menu.html',
-        'appMenuItems': [
-          {
-            'label': 'MyUW home',
-            'icon': 'home',
-            'url': '/web',
-          },
-        ],
+        'enablePushContentMenu': true,
+      },
+      'SERVICE_LOC': {
+        'messagesURL': 'staticFeeds/sample-messages.json',
```

Comment:
Maybe the ootb `override.js` should not override the default messages-are-off `messagesURL` to keep this feature out of the way of local app development (which most of the time won't be developing messages content).